rasa/nlu/tokenizers/jieba_tokenizer.py

```diff
@@ -75,25 +103,25 @@ def tokenize(self, message: Message, attribute: Text) -> List[Token]:
     @classmethod
     def load(
         cls,
-        meta: Dict[Text, Any],
-        model_dir: Text,
-        model_metadata: Optional["Metadata"] = None,
-        cached_component: Optional[Component] = None,
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
         **kwargs: Any,
-    ) -> "JiebaTokenizer":
-        """Loads trained component (see parent class for full docstring)."""
-        relative_dictionary_path = meta.get("dictionary_path")
+    ) -> JiebaTokenizerGraphComponent:
+        """Loads a custom dictionary from model storage."""
+        relative_dictionary_path = config["dictionary_path"]
```

Comment:
what guarantees that this is relative? I think we should be able to handle absolute paths as well (e.g. just loading everything in the `resource_directory` instead of doing weird name stuff)

Reply:
This is just what they called it before. It's not really relative, it's just not in the model storage. If the dictionary path is set in the config it means they are using a custom dictionary and we would have saved it to the model storage. 
However we could also just always try to load whatever is in the model storage.

Reply:
> However we could also just always try to load whatever is in the model storage.
 Write

I'd just do that as we'd only move things to the model storage which were part of `dictionary_path`

Reply:
However if there was no `dictionary_path` in the config we won't have created the resource so we will get a `ValueError`. And this error could signify either we forgot to persist, or we just didn't use custom dictionaries. Whereas if we keep this check, we know something is wrong if there is a custom dictionary path _and_ no resource.